packages/core/src/__tests__/ceramic-anchor.test.ts

```diff
@@ -311,7 +311,7 @@ describe('Ceramic anchoring', () => {
     expect(doctype1.content).toEqual({ x: 6 })
     expect(doctype1.state.log.length).toEqual(8)
 
-    const doctype2 = await ceramic2.loadDocument(doctype1.id)
+    const doctype2 = await ceramic2.loadDocument(doctype1.id, {sync: true})
```

Comment:
Isn't this the default value for this option?

Reply:
It is indeed.

Reply:
So does this really solve the problem?

Reply:
Not at all.